### DISCUSSION TITLE
git config in Github Action

### DISCUSSION THREAD

**Discussion by tasty0tomato** [zhongfly/mpv-build#14] [Q&A] [answered]
I also wanted to build mpv using Actions, but I read all documents about Github Actions and I could not handle the `git config`. It seemed there was some limits.
Do you know where is it mentioned in the documents?
Thanks

**Comment by zhongfly** [comment, marked as answer]
Can you give an example and show me the log?
or try using sudo

**Reply by tasty0tomato** [reply]
For example，in your repo
```
git config --global user.name "github-actions[bot]"
git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

**Reply by zhongfly** [reply]
![Screenshot_20220131111615](https://user-images.githubusercontent.com/11155705/151734253-87feeed1-eaf6-4a11-9a01-cfae1a8defae.png)

I would recommend you to go to https://github.community/ for help
